package.json

```diff
@@ -58,15 +58,19 @@
   },
   "dependencies": {
     "3d-view": "^2.0.0",
-    "@plotly/d3-sankey": "^0.5.1",
     "alpha-shape": "^1.0.0",
     "array-range": "^1.0.1",
     "canvas-fit": "^1.5.0",
     "color-normalize": "^1.3.0",
     "convex-hull": "^1.0.3",
     "country-regex": "^1.1.0",
     "d3": "^3.5.12",
+    "d3-sankey": "git://github.com/antoinerg/d3-sankey.git#4f37ed8d3578b545a8569ecd74583f373768e900",
```

Comment:
Minimal changes over latest d3-sankey https://github.com/d3/d3-sankey/compare/master...antoinerg:fix-large-padding

Reply:
Hmm. So we no longer need https://github.com/plotly/d3-sankey ? I thought [i read](https://github.com/plotly/plotly.js/issues/2636#issuecomment-389330558) that we made changes in our `@plotly/d3-sankey` that got rejected by `d3/d3-sankey`.

Reply:
Those changes were minimal. One of the biggest was that we generated the SVG path for the links in it. As you found out in this PR, we can do operation in plotly.js instead.

Reply:
Ok that make sense. Do you think we have a shot at getting

https://github.com/d3/d3-sankey/compare/master...antoinerg:fix-large-padding

merged into `d3/d3-sankey`?

Reply:
There hasn't been anything merged into `master` in `d3/d3-sankey` since Jul 13, 2017 so I'm not sure... I opened a PR almost a month ago https://github.com/d3/d3-sankey/pull/63 and there has been no activity since then.